source/adapters/level_zero/device.cpp

```diff
@@ -919,6 +919,31 @@ UR_APIEXPORT ur_result_t UR_APICALL urDeviceGetInfo(
     return ReturnValue(true);
   case UR_DEVICE_INFO_COMMAND_BUFFER_UPDATE_SUPPORT_EXP:
     return ReturnValue(false);
+  case UR_DEVICE_INFO_BINDLESS_IMAGES_SUPPORT_EXP:
+    return ReturnValue(true);
+  case UR_DEVICE_INFO_BINDLESS_IMAGES_SHARED_USM_SUPPORT_EXP:
+    return ReturnValue(true);
+  case UR_DEVICE_INFO_BINDLESS_IMAGES_1D_USM_SUPPORT_EXP:
```

Comment:
ZE experimental bindless image is only defined for 2D image - this is what cuda support - and we didnt decide to support 1D.

Reply:
thanks